.github/workflows/main.yml

```diff
@@ -148,58 +183,19 @@ jobs:
           prerelease: false
           commitish: master
           body: |
-            | Binary        | SHA256        |
+            | Binary        | Download link |
             | ------------- |:-------------:|
-            | OctoBot_windows_x64.exe | ${{ steps.hashes.outputs.octobot_windows_x64_hash }} |
-            | OctoBot_linux_x64 | ${{ steps.hashes.outputs.octobot_linux_x64_hash }} |
-            | OctoBot_linux_arm64 | ${{ steps.hashes.outputs.octobot_linux_arm64_hash }} |
-            | OctoBot_macos_x64 | ${{ steps.hashes.outputs.octobot_macos_x64_hash }} |
-
-      - name: Upload Release Asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.AUTH_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ./OctoBot_windows_x64.exe/OctoBot_windows.exe
-          asset_name: OctoBot_windows_x64.exe
-          asset_content_type: application/x-binary
-
-      - name: Upload Release Asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.AUTH_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: OctoBot_linux_x64/OctoBot_x64
-          asset_name: OctoBot_linux_x64
-          asset_content_type: application/x-binary
-
-      - name: Upload Release Asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.AUTH_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: OctoBot_linux_arm64/OctoBot_arm64
-          asset_name: OctoBot_linux_arm64
-          asset_content_type: application/x-binary
-
-      - name: Upload Release Asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.AUTH_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: OctoBot_macos_x64/OctoBot_x64
-          asset_name: OctoBot_macos_x64
-          asset_content_type: application/x-binary
+            | OctoBot_windows_x64.exe | ${{ secrets.RELEASE_BASE_URL }}/${{ steps.vars.outputs.tag }}/OctoBot_windows_x64.exe |
+            | OctoBot_linux_x64 | ${{ secrets.RELEASE_BASE_URL }}/${{ steps.vars.outputs.tag }}/OctoBot_linux_x64 |
+            | OctoBot_linux_arm64 | ${{ secrets.RELEASE_BASE_URL }}/${{ steps.vars.outputs.tag }}/OctoBot_linux_arm64 |
+            | OctoBot_macos_x64 | ${{ secrets.RELEASE_BASE_URL }}/${{ steps.vars.outputs.tag }}/OctoBot_macos_x64 |
```

Comment:
:+1:

Reply:
should we include hashes as well ?

Reply:
I don't know. I think it would be better to upload them to make them available for anyone using release.octobot.online

Reply:
upload hashes values and display them in release.octobot.online ? 👍

Reply:
Yes